package-lock.json

```diff
@@ -2086,7 +2086,6 @@
         "anymatch": "^2.0.0",
         "async-each": "^1.0.1",
         "braces": "^2.3.2",
-        "fsevents": "^1.2.7",
```

Comment:
Are the changes in this file intentional?

Reply:
No there were not intentional, I've removed this Thanks!

Reply:
No, they were not has been reverted Thanks!